babel/src/board.js

```diff
@@ -247,14 +244,14 @@ export class Board {
           * 3. if the row isn't 0, it takes the orb from above
           * 4. if the row is 0, it creates a random orb
         */
-        _.each(_.range(this.height), x =>{
-            _.each(_.range(this.width), y => { //1
-                if (this.orbs[x][y] == 'X') {
-                    for (var z = x; z >= 0; z--) { //2
+        _.each(_.range(this.height), row =>{
+            _.each(_.range(this.width), col => { //1
+                if (this.orbs[row][col] == 'X') {
```

Comment:
Right now, I am thinking of each time finding a number or symbol and making sure it is not a possible orb. So what gets dropped in is different each time. Something like take the first number from 0-100, or all the alphabet that isn't in `this.types`. Then it just has to assume someone doesn't want a gigantic board with 150 different types.